esrally/mechanic/supplier.py

```diff
@@ -49,9 +49,14 @@ def create(cfg, sources, distribution, build, car, plugins=None):
     template_renderer = TemplateRenderer(version=es_version, os_name=target_os, arch=target_arch)
 
     if build_needed:
-        java_home = _java_home(car)
+        raw_build_jdk = car.mandatory_var("build.jdk")
+        try:
+            build_jdk = int(raw_build_jdk)
+        except ValueError:
+            raise exceptions.SystemSetupError(f"Car config key \"build.jdk\" is invalid: \"{raw_build_jdk}\" (must be int)")
```

Comment:
FWIW im perfectly fine if we use single quotes inside f strings for things like this... `f"blah 'this thing' blah"`, I hate seeing `\` when we can get around it :)

If you dont agree, thats fine by me!

Reply:
Good point. Usually we use square brackets anyway instead of double quotes. I've pushed a change in https://github.com/elastic/rally/pull/1027/commits/690d568ccd4f148d3789712d143b9276bf433053.